i18n/overrides/en.json

```diff
@@ -45,5 +45,14 @@
+	"createwiki-close-email-sender": "Miraheze",
+	"createwiki-email-subject": "Your new Miraheze wiki: $1",
+	"createwiki-email-body": "Hello!\n\nYou recently requested a wiki on Miraheze. We are pleased to inform you that it has been approved!\n\nMiraheze is here to help you every step of the way, don't be afraid to ask! Visit our <a href=\"https://meta.miraheze.org/wiki/Special:MyLanguage/Help_center\">Help center</a> for detailed help on a range of topics, such as managing your wiki using our ManageWiki tool, using images and templates, filing tasks, and more!\n\nPlease remember that all wikis must abide by the <a href=\"https://meta.miraheze.org/wiki/Special:MyLanguage/Content_Policy\">Content Policy</a> or they may be closed. Remember that your wiki cannot deviate too much from it's approved purpose or it may be closed too.\n\nThank you for choosing Miraheze! We wish you all the best on your new wiki adventure.",
+	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again, and if the problem persists [[Help center|please contact us]]",
+	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and one of our wiki creators should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
+	"requestwiki-header": "Thank you for choosing Miraheze to host your wiki! To get started, please fill out this form so we can review your request.\nYour wiki request will be reviewed by a volunteer, and it can be tracked at the page that will be given when submitting your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first. You must first choose a subdomain, and your custom domain will be configured afterwards. \n\nWhen you request a wiki, you automatically agree to our [[Content Policy|Content]] and [[Dormancy Policy]].",
+	"requestwiki-label-siteurl": "Subdomain: (subdomain.miraheze.org)",
```

Comment:
This one in particular I think can be left in CreateWiki. I intend to make the domain example be a `$1` value later, that is based on domain configuration.

Reply:
Done